modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Kresimir Coko
+ */
+public class MultiSelectTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		return super.doStartTag();
+	}
+
+	public String getClearAllTitle() {
+		return _clearAllTitle;
+	}
+
+	public String getCloseButtonAriaLabel() {
+		return _closeButtonAriaLabel;
+	}
+
+	public boolean getDisabledClearAll() {
+		return _disabledClearAll;
+	}
+
+	public Object getFilter() {
+		return _filter;
+	}
+
+	public String getInputName() {
+		return _inputName;
+	}
+
+	public String getInputValue() {
+		return _inputValue;
+	}
+
+	public boolean getIsValid() {
```

Comment:
what is valid?

Reply:
@carlos The input, taken from the Clay docs. I assume you meant it more like changing the name to `getIsInputValid`?

Reply:
@kresimir-coko I think you meant to mention @carloslancha

Reply:
Yea thanks @julien

Reply:
Yep, I meant the name, `isValid` says nothing to me. What is valid? Taking a look to Clay code:

```
className={classNames(
	'form-control form-control-tag-group input-group',
	{
		focus: isFocused && isValid,
	}
)}
```

That's the only place where `isValid` is used. So... we're adding the class `focus` if `isValid` is `true`? I probably expect an error to be shown if `isValid` is false or something, this param makes no much sense to me. You can ask Clay guys for more context on this topic and propose a change if needed.